File: src/main/java/io/dustin/finstage/financial/domain/FinancialStatement.java
package io.dustin.finstage.financial.domain;

import lombok.Builder;
import lombok.Getter;

import java.util.List;

@Getter
@Builder
public class FinancialStatement {
    private final String symbol;
    private final List<IncomeStatement> incomeStatements;
    private final List<BalanceSheet> balanceSheets;
    private final List<CashFlowStatement> cashFlowStatements;
}

File: src/main/kotlin/io/dustin/finstage/financial/domain/FinancialStatement.kt
package io.dustin.finstage.financial.domain

data class FinancialStatement(
        val symbol: String,
        val incomeStatements: List<IncomeStatement>,
        val balanceSheets: List<BalanceSheet>,
        val cashFlowStatements: List<CashFlowStatement>
)
